### PR TITLE
ssl config support for postgres

### LINKED ISSUE #16: PostgreSQL SSL Support
Following up on the [sqlectron-gui issue to add SSL mode](https://github.com/sqlectron/sqlectron-gui/issues/72), I found I was still unable to connect to my Heroku psql db. 

sqlectron-ssl

Note that error states SSL is off, despite state of "SSL Mode" switch.  Digging into the code, I found the problem was with sqlectron-core. While MySQL and SQLServer both adopt ssl settings from server.config, PostgreSQL does not. 

I am submitting a pull request with the fix, and while I have built sqlectron-gui and tested that it does in fact work, I did not write specs as there were none for the already implemented mysql and sqlserver ssl support.

### PR DESCRIPTION
**Pull request by ckee83**
fixes #16

### CODE REVIEW

File: src/db/clients/postgresql.js [hunk 1]
```diff
@@ -291,6 +291,8 @@ function _configDatabase(server, database) {
     config.port = server.config.localPort;
   }
 
+  if (server.config.ssl) config.ssl = server.config.ssl;
```
Comment (maxcnunes) [inline src/db/clients/postgresql.js]: Could just break the if statement into a new line with curly brackets? Just to keep the code with the same pattern.

**Comment by BornaP** [comment]
Thanks for the pull request @ckee83 ! :clap: :+1:

I just squashed commits so the history looks cleaner, since second and third commit were just syntactical changes.

**Comment by ckee83** [comment]
Fantastic! Thanks guys :+1:

**Comment by paulomcnally** [comment]
What is version implemented? My version is 1.8.0

Thanks

**Comment by maxcnunes** [comment]
It was implemented after 1.8.0 was released. I am planning in release a new version in the next week.

**Comment by paulomcnally** [comment]
Thanks for you're response @maxcnunes 

How to compile from github to obtain the most recent changes?

**Comment by maxcnunes** [comment]
Following these steps https://github.com/sqlectron/sqlectron-gui#build.

Although we have just updated the electron to the latest version and seems the SSH tunnel is not working properly. I trying to figured out whats is the problem.

**Pull request merged**